howto/descriptor.po

```diff
@@ -70,6 +79,11 @@ msgid ""
 "`__delete__`.  If any of those methods are defined for an object, it is said "
 "to be a descriptor."
 msgstr ""
+"En général, un descripteur est un attribut objet avec un \"comportement "
```

Comment:
un attribut d'un objet, dont l'accès est régi par des méthodes respectant le protocole "descripteur", ce qui induit un comportement lié à ces méthodes.

Reply:
Excuse me I did not understand well. What sentence would you put in French?

Reply:
En général, un descripteur est un attribut d'un objet, dont l'accès est régi par des méthodes respectant le protocole "descripteur", ce qui induit un comportement lié à ces méthodes.